docs/rdl/index.md

```diff
@@ -0,0 +1,20 @@
+# Overview
```

Comment:
There's a lot of overlap between this page and `docs/index.md`. Recommend integrating the two pages into a single index page.

Reply:
updated